packages/shared_preferences/shared_preferences_android/android/src/main/java/io/flutter/plugins/sharedpreferences/MethodCallHandlerImpl.java

```diff
@@ -22,17 +18,12 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.LinkedBlockingQueue;
-import java.util.concurrent.ThreadPoolExecutor;
-import java.util.concurrent.TimeUnit;
-
-/**
- * Implementation of the {@link MethodChannel.MethodCallHandler} for the plugin. It is also
- * responsible of managing the {@link android.content.SharedPreferences}.
- */
+
+/** Helper class to save data to `android.content.SharedPreferences` */
+
+// Rename class and file to match it's purpose, preferably SharedPreferencesHelper
```

Comment:
If you would still like to make this change, I think another pr that does only this would be a good idea.

Reply:
Should I change the name perhaps?

Reply:
I think both in a follow up pr would be ideal

Reply:
Yeah sure, do I have to do it after this PR is merged? I am sorry for the lame question😔, this is my first time contributing.

Reply:
you can put the pr up whenever, I think it would be easier to wait till this one is landed though

Reply:
Okay sure! Thanks a lot again(🙂).